src/aaudio/AAudioLoader.cpp

```diff
@@ -38,15 +38,18 @@ int AAudioLoader::open() {
     if (mLibHandle != nullptr) {
         return 0;
     }
-    mLibHandle = dlopen(LIB_AAUDIO_NAME, 0);
+    mLibHandle = dlopen(LIB_AAUDIO_NAME, 0);  // RTLD_LAZY instead of 0?
     if (mLibHandle == nullptr) {
         LOGI("AAudioLoader::open() could not find " LIB_AAUDIO_NAME);
         return -1; // TODO review return code
     } else {
         LOGD("AAudioLoader():  dlopen(%s) returned %p", LIB_AAUDIO_NAME, mLibHandle);
     }
 
-    // Load all the function pointers.
+    // Load all the function pointers. - Why Some of the pointers loaded by
+    // calling `dlsym` directly, while the others use helper functions like
+    // load_V_PBI or load_I_PS? Those functions also call `AAudioLoader_check`
```

Comment:
Because some function signatures only have one occurrence. But I can make this change.